### ISSUE TITLE
No image.

### ISSUE THREAD

**Issue by hack-everything** [Aryan-Kanak/driving-simulator#1]
self.sprite = pygame.image.load(img_filepath).convert_alpha() # filepath for sprite of car

**Comment by PycraftDeveloper** [comment]
I believe @hack-everything that you need to give Pygame more information as to the location of the picture that you want to load in "pygame.image.load()", could you please give some more information as to the resources that that game uses to we can help more with this issue as I believe you need to specify the a sub-folder in which the sprite is found. but besides from that the code looks perfect!

**Comment by PycraftDeveloper** [comment]
Yes indeed it does lol, thank you @hack-everything, I hope you have a good day this is very helpful, it does need actual images.
